Pipfile

```diff
@@ -11,7 +11,7 @@ scikit-learn = "*"
 
 [packages]
 numpy = "*"
-torch = "~=1.4"
+torch = "==1.4"
```

Comment:
Why this change?

Reply:
Dataparallel is buggy in torch 1.5 (https://github.com/huggingface/transformers/issues/3936)
The only known fix so far is reverting back to 1.4